scripts/decredStakeVoting.md

```diff
@@ -9,9 +9,7 @@ To participate in the proof-of-stake mining system, users lock a portion of thei
 
 Tickets are chosen to vote randomly using arbitrary entropy generated by the blockchain.
 
-A tickets votes are cast by a user chosen stake server.
-
-Stake servers stand by 24/7 to ensure votes are counted and rewards are earned, serving to eliminate losses due to computer downtime or bad Internet connections. In return, stake servers receive a small portion of the proof-of-stake rewards to help cover operational costs.
+When a ticket is called to vote, it must respond quickly. It is possible to solo stake tickets but solo stakers usually maintain a number of servers to ensure up-time. Many ticket holders use instead stakepools for this purpose. Stakepools collect a small share of the voting reward in exchange for providing the infrastructure to ensure reliable voting. The use of stakepools is integrated in the Decrediton wallet, making it easy to vote this way.
```

Comment:
Good information but may be too verbose. We chose to use the term 'stake server' to avoid confusion with 'ticket pool'